binnacle/lib/services/bluetooth_parser.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:sos/models/compass_model.dart';
+import 'package:sos/models/position_model.dart';
+import 'package:sos/models/wind_model.dart';
+import '../enums.dart';
+
+/*
```

Comment:
We should add a TODO: update packet format annotation

Reply:
Also, dart style uses /// comments for documentation